src/build/buildIcon.ts

```diff
@@ -83,12 +83,16 @@ export function convertIconIfNecessary(options: AppOptions): void {
   }
 
   try {
-    const iconPath = convertToIcns(options.packager.icon);
-    options.packager.icon = iconPath;
-    return;
+    if (!iconIsIcns(options.packager.icon)) {
+      const iconPath = convertToIcns(options.packager.icon);
+      options.packager.icon = iconPath;
+    }
+    if (options.nativefier.tray) {
+      convertToTrayIcon(options.packager.icon);
+    }
   } catch (error) {
     log.warn('Failed to convert icon to .icns, skipping.', error);
     options.packager.icon = undefined;
-    return;
   }
+  return;
```

Comment:
With this function being void, I don't think you actually need a return here.

Reply:
yup